Src/Managers/event_manager.py

```
@@ -1,81 +1,115 @@
+        self.__global_calls[event_type].append(handler)
+        callback = self.__get_callback(self.__global_calls[event_type])
+        event_type(parent=global_registry, callback=callback, user_data=user_data)
+
+        self.__logger.info(f"Глобальный обработчик '{event_type}' добавлен")
+        self.__logger.debug(f"Нынешние обработчики: {self.__global_calls[event_type]}")
+
+
+    def clear(self, item_id: str | int):
+        """Удаляет все обработчики, привязанные к элементу."""
+        if not dpg.does_item_exist(item_id):
+            self.__logger.warn(f"Попытка удаления обработчиков с ({item_id}), которого не существует")
```

Ruff (0.15.21)

[warning] String contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF001)